sticht/rollbacks/base.py

```diff
@@ -0,0 +1,235 @@
+import abc
+from typing import Any
+from typing import List
+from typing import Optional
+from typing import Tuple
+
+from sticht.rollbacks.slo import SLOWatcher
+from sticht.rollbacks.slo import watch_slos_for_service
+from sticht.slack import Emoji
+from sticht.slack import SlackDeploymentProcess
+
+
+class RollbackSlackDeploymentProcess(SlackDeploymentProcess, abc.ABC):
```

Comment:
this is basically a rename of SLOSlackDeploymentProcess with some differences i'll call out below